src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -45,23 +43,24 @@ export default class ReplyField extends React.Component<Props, State> {
         value: '',
     };
 
-    constructor(props: Props) {
-        super(props);
+    editorRef: React.MutableRefObject<Editor | null> = React.createRef<Editor | null>();
```

Comment:
Is the type declaration needed? And is the ref used anywhere?

Reply:
Good catch